src/default_path.rs

```diff
@@ -0,0 +1,10 @@
+extern crate mozrunner;
+
+use mozrunner::runner::platform;
+
+fn main() {
+    let path = platform::firefox_default_path()
+        .and_then(|x| x.into_os_string().into_string().ok())
+        .unwrap_or("Firefox binary not found".to_owned());
+    println!("{}", path);
+}
```

Comment:
I think this would be nicer:

```rust
    match platform::firefox_default_path() {
        Some(p) => println!("{}", p.to_string_lossy().into_owned()),
        None => std::process::exit(1),
    };
```